### PR TITLE
doc and images for filestorecsi images 1.5.0, 1.4.4, 1.3.13

### PR DESCRIPTION
**Pull request by saikat-royc**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
/kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
none
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,9 +54,11 @@ The following table captures the compatibility matrix of the core filestore driv
 | v1.3.10 (GA)                            |  yes |  yes  |
 | v1.3.11 (GA)                            |  yes |  yes  |
 | v1.3.12 (GA)                            |  yes |  yes  |
+| v1.3.13 (GA)                            |  yes |  yes  |
 | v1.4.1 (GA)                             |  yes |  yes  |
 | v1.4.2 (GA)                             |  yes |  yes  |
 | v1.4.3 (GA)                             |  yes |  yes  |
+| v1.4.4 (GA)                             |  yes |  yes  |
```
Comment (leiyiz) [inline README.md:61]: should we also add 1.5.0 here?

Comment (saikat-royc) [reply README.md:61]: yes. Added

**Pull request merged**